src/terminal/Screen.h

```diff
@@ -485,11 +485,7 @@ class Screen final: public ScreenBase, public capabilities::StaticDatabase
 
     [[nodiscard]] bool compareCellTextAt(CellLocation position, char codepoint) const noexcept override
     {
-        return grid()
-            .lineAt(position.line)
-            .inflatedBuffer()
-            .at(position.column.as<size_t>())
-            .compareText(codepoint);
+        return grid().lineAt(position.line).compareCellTextAt(position.column, codepoint);
```

Comment:
I think - with the above comment - the old code part here could still be used instead (no need for a special function unless an optimized implementation could be easily found). What do you think?

Reply:
I changed it to be like previous. Maybe we can think about it later :)